src/lib.rs

```diff
@@ -3,6 +3,8 @@
 
 mod web_asset_io;
 mod web_asset_plugin;
+mod web_filesystem_watcher;
```

Comment:
i think the web prefix here is slightly confusing. Maybe `local_filesystem_watcher` or just `filesystem_watcher`?